doc/api/deprecations.md

```diff
@@ -2247,6 +2247,22 @@ Type: Documentation-only (supports [`--pending-deprecation`][])
 Directly calling `process.binding('uv').errname(<val>)` is deprecated.
 Please make sure to use [`util.getSystemErrorName()`][] instead.
 
+<a id="DEP00XX"></a>
+### DEP00XX: net._setSimultaneousAccepts()
+<!-- YAML
+changes:
+  - version: REPLACEME
+    pr-url: TODO
+    description: Runtime deprecation
+-->
+
+Type: Runtime
+
+The `net._setSimultaneousAccepts()` function is an undocumented originally
```

Comment:
I may be wrong, but this sentence seems a bit too complicated (too long and oversaturated?). Maybe we can split it?

Reply:
Also "an undocumented originally" - missing a word.

Reply:
```suggestion
The undocumented `net._setSimultaneousAccepts()` function was originally
```